### PR TITLE
feat: esm

### PR DESCRIPTION
**Pull request by ThaUnknown**
**What is the purpose of this pull request? (put an "X" next to item)**

- [x] Documentation update
- [ ] Bug fix
- [x] New feature
- [ ] Other, please explain:

**What changes did you make? (Give an overview)**

- chore: update docs and package description
- refactor: switch to esm imports and exports

**Which issue (if any) does this pull request address?**
- https://github.com/webtorrent/webtorrent/issues/2177

**Is there anything you'd like reviewers to focus on?**

### CODE REVIEW

File: client.js [hunk 1]
```diff
@@ -68,7 +70,7 @@ class Client extends EventEmitter {
     // Remove trailing slash from trackers to catch duplicates
     announce = announce.map(announceUrl => {
       announceUrl = announceUrl.toString()
-      if (announceUrl[announceUrl.length - 1] === '/') {
+      if (announceUrl[announceUrl.length - 1] === '/index.js') {
```
Comment (DiegoRBaquero) [inline client.js:73]: Is this an automated change?

**Review by DiegoRBaquero — COMMENTED** [review]
Rest of things look good

File: client.js (hunk 1 above)
Comment (ThaUnknown) [reply client.js:73]: yeah, linter auto-fixed that, didn't even notice

Comment (ThaUnknown) [reply client.js]: @DiegoRBaquero fixed

**Review by DiegoRBaquero — APPROVED** [review]

**Pull request merged**